publications-2021.bib

```diff
@@ -548,14 +548,14 @@ @Article{kourakos.harter:simulation
 @InProceedings{kronbichler.fehn.ea:next-generation,
   author  = {Kronbichler, Martin and Fehn, Niklas and Munch, Peter and Bergbauer, Maximilian and Wichmann, Karl-Robert and Geitner, Carolin and Allalen, Momme and Schulz, Martin and Wall, Wolfgang A.},
   title   = {A Next-Generation Discontinuous {G}alerkin Fluid Dynamics Solver with Application to High-Resolution Lung Airflow Simulations},
-  year    = {2021},
-  isbn    = {9781450384421},
+  year    = 2021,
+  isbn    = 9781450384421,
```

Comment:
Hmm, this change also adjusts `isbn` numbers. I do not know how `bibtex` reacts to this.

Reply:
Bibtex sees a number and parses it correctly. Bibtex doesn't care what the number *means*.